modular_skyrat/modules/modular_slots/code/human.dm

```diff
@@ -0,0 +1,46 @@
+/mob/living/carbon/human
+	// The actual slot.
+	/// The passport the human has in their passport slot.
+	var/obj/item/passport
+
+// Clean up your slots when a human mob is destroyed here.
+/mob/living/carbon/human/Destroy()
+	. = ..()
+	qdel(passport)
```

Comment:
```suggestion
	if(passport)
		QDEL_NULL(passport)
```

Reply:
Isn't done this way in supercode, but I'll do it.